PaulasWeatherApp/PaulasWeatherApp/ViewModel/WeatherViewModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  WeatherViewModel.swift
+//  PaulasWeatherApp
+//
+//  Created by Stef, Paula on 8/19/22.
+//
+
+import Foundation
+
+class WeatherViewModel: NSObject {
+    private var weatherService: WeatherService?
+    private let defaults = UserDefaults.standard
+    private(set) var weatherData: WeatherDataModel? {
+        didSet {
+            weatherDataUpdated()
+        }
+    }
+    var weatherDataUpdated : (() -> ()) = {}
+    
+    override init() {
+        super.init()
+        weatherService = WeatherService()
+    }
+    //MARK: - Get data methods
+    func getWeatherData() {
+        let time = Int(NSDate().timeIntervalSince1970 - 100)
```

Comment:
where is the 100 coming from? if you need it, add a constant that says what does it mean